config/default.yml

```diff
@@ -107,6 +107,13 @@ Sorbet/SignatureBuildOrder:
   Enabled: true
   VersionAdded: 0.3.0
 
+Sorbet/SingleLineRbiClassModuleDefinitions:
+  Description: 'Empty class and module definitions in RBI must be on a single line.'
+  Enabled: true
```

Comment:
Can we have this disabled by default? This is the style we prefer at Shopify but it might not be the style others would like, so I'd rather this be an opt-in for the library.

Reply:
Makes sense! Will do.